distributed/client.py

```diff
@@ -937,6 +942,11 @@ def __init__(
 
         ReplayTaskClient(self)
 
+    def _set_default_configs(self):
+        self._set_config_stack.push(
```

Comment:
the `dask.config.set.__enter__` is a noop so `push` and `enter_context` do the same thing - but I think it's clearer to use `enter_context` anyway

```suggestion
        self._set_config_stack.enter_context(
```